src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -87,14 +103,17 @@ protected internal override void Setup(FeatureConfigurationContext context)
             }
         }
 
-        static void ImportMessageEndpointMappings(FeatureConfigurationContext context, MessageEndpointMappingCollection legacyRoutingConfig, TransportInfrastructure transportInfrastructure, Publishers publishers, UnicastRoutingTable unicastRoutingTable)
+        static Type[] GetKnownMessageTypes(FeatureConfigurationContext context)
```

Comment:
I just checked. We are currently using GetAvailableTypes() in 14 different locations in core. Most of them filter with Conventions.IsMessageType. Raised an issue https://github.com/Particular/NServiceBus/issues/3947